### ISSUE TITLE
AutoComplete - The onSelect event is emitted twice

### ISSUE THREAD

**Issue by moacir-junior** [primefaces/primeng#10387]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
The onSelect event is emitted twice.

**Current behavior**
When an item is selected in the Autocomplete field, the onSelect event is emitted twice. This behavior happens right after leaving the field.
Without the forceSelection option the event is correctly emitted.
 
**Expected behavior**
The onSelect event should be emitted only once.

**Minimal reproduction of the problem with instructions**
https://github.com/moacir-junior/PrimeNG-Autocomplete
The problem can be reproduced in the link above.
(Select a country in the autocomplete field. The content in the console will be emitted twice.)

 **What is the motivation / use case for changing the behavior?**
The event should be emitted only once so that validations can be correctly performed.

 **Angular version:**
9.1.13. Happens in version 12.0.3.

 **PrimeNG version:**
9.1.13. Happens in version 12.0.3.

**Browser:**
Google Chrome v91.0.4472.124
Firefox v88.0.1
Microsoft Edge v91.0.864.64

**Comment by cetincakiroglu** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
